include/caf/sensor_sampler.rst

```diff
@@ -0,0 +1,220 @@
+.. _caf_sensor_sampler:
+
+CAF: Sensor sampler module
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_sampler| of the :ref:`lib_caf` (CAF) generates the following types of events in relation with the sensor defined in the module configuration:
+
+* ``sensor_event`` when the sensor is sampled.
+* ``sensor_state_event`` when the sensor state changes.
+
+Configuration
+*************
+
+To use the module, you must complete the following requirements:
+
+1. Physically connect the device and add and enable it in the :file:`BOARD.dts` devicetree file.
```

Comment:
Its common practice. Done similarly in other places like Zephyr's devicetree guide. This is common application framework not application specific documentation.

Reply:
I would say that it's just one of the options. Maybe state that you can do it `for example in BOARD.dts file`.
NCS applications may link directly here and I am aftraid that it could cause confusion in case we support multiple build types and `dts.overlay` file is defined in other way.

Reply:
``` suggestion
1. Physically connect the sensor, add and enable it, for example, in the :file:`BOARD.dts` devicetree file.

```

Is that fine?